add note in pages functions docs clarifying that not all bindings are supported (#14353)
cloudflare/cloudflare-docs: content/pages/functions/bindings.md

This guide will instruct you on configuring a binding for your Pages Function. You must already have a Cloudflare Developer Platform resource set up to continue.

{{<Aside>}}

> Note that Pages Functions does not support all [binding](/workers/runtime-apis/bindings/) but a subset of them and that only the bindings noted in this page are officially supported by Pages Functions.

Author: 🤦👍

> Note that Pages Functions does not support all [bindings](/workers/runtime-apis/bindings/) but a subset of them and that only the bindings noted in this page are officially supported by Pages Functions.

Reviewer: ```suggestion
Pages Functions only support a subset of all [bindings](/workers/runtime-apis/bindings/), which are listed on this page.
```
